optax/_src/loss_test.py

```diff
@@ -475,5 +475,20 @@ def test_batched(self):
         self.exp,
         atol=1e-4)
 
+class HingeLossTest(parameterized.TestCase):
+
+  def setUp(self):
+    super().setUp()
+    self.ys = np.array([1, 0, 1, 0, 0, 1, 0, 0, 1, 0])
```

Comment:
Do I understand correctly that `ys`, the predictor output can be in the range `(-Inf, Inf)`? I think it would be important to test this, also as a way of documenting the function. In particular, I think your test don't test the `maximum` behaviour in that they would also pass for the implementation `return delta - predictor_outputs * targets`.

Reply:
Thanks for the review! From what I understand by this is that I should add a different example here that would test the maximum behavior?

Reply:
Hi @mkunesch, can you please clarify on this?

Reply:
Yes, please - you could just add some numbers here for which the maximum behaviour is important. Also, currently the ys are only 0 and 1 but if I understand correctly they can be any float in the range (-Inf, Inf) in practice so the test input should reflect this.